tools/setlangcookie.php

```diff
@@ -12,7 +12,12 @@
 if($language)
 {
     // set the cookie
-    setcookie("language", $language, time()+31536000, "/");
+    setcookie("language", $language, [
```

Comment:
I am not sure of this scenario, but it looks like you can set the language with a "lang" query param which results in a cookie being set.